doc/renaming_table.xml

```diff
@@ -0,0 +1,30 @@
+<Table Align="|l|l|">
```

Comment:
This file is an automatically generated file so it must not be added to the commit. You can do `git rm doc/renaming_table.xml` to remove it.